Common/src/geometry/CPhysicalGeometry.cpp

```diff
@@ -7206,80 +7206,56 @@ void CPhysicalGeometry::GatherInOutAverageValues(CConfig *config, bool allocate)
 
 
 void CPhysicalGeometry::SetCoord_CG(void) {
-  unsigned short nNode, iDim, iMarker, iNode;
+  unsigned short iMarker, iNode;
   unsigned long elem_poin, edge_poin, iElem, iEdge;
-  su2double **Coord;
+
+  /// TODO: What is a reasonable MAXNNODE?
```

Comment:
Is that a todo that you still want to resolve? Otherwise I would say that 64 is definitely enough.

Reply:
Oh I forgot that, I think 64 is enough too but if anyone as a definitive answer based on the element types we handle... I don't know how crazy a dual volume can get in terms of how many edges connect to it. (@economon any idea?)

Reply:
But its just based on the primal structure, isn't it? So the maximum would be 8 for a hexahedron.

Reply:
Duh! Your are absolutely right, thanks. I'll fix this, but don't hold the release back because of this PR, this has quite a few changes and we had a ton of small bug fixes, it would be a shame if this introduces some.

Reply:
Alright. Then I'll go ahead :)